python/genhisto.py

```diff
@@ -55,13 +56,26 @@ def main():
 
         get_minmax_out = None
 
+        #TODO: use temp file or overwrite input file instead of using a hardcoded filename and send to minmaxmap.py as argument
+        filename_file = 'filenames_updated.txt'
+        hdfs_file = None        
+
+        #updated file containing filepaths to nifti images will be located on hdfs
+        if 'file:///' not in input_file:
+            filename_file = client.root + filename_file
```

Comment:
hum, so now we *can't* pass HDFS URLs using the hdfs URL protocol but we have to pass paths relatively to `client.root`, right? That's not ideal but I understand that it may be coming from the HDFS lib (although I find it strange). At least this should be documented properly.